### ISSUE TITLE
Calendar Events for Webinars and Office Hours

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos-webinars#50]
@ebraker At some point the default event visibility for the Arctos Working Group Calendar was changed to Private. This means that any event created for public consumption needs to have the visibility changed to Public if we want random people to be able to see the event details. I just did this for the office hours next week.

But also, looking at that particular event, it only has one invitee - Cynthia Carr, our former intern. Without an invite, I wonder how many people actually know about the office hour? Should we just add Arctos Working Group Google Group to the invite list? This would send a notification to everyone in the Google Group about the office hour. Will this become too much Arctos email or will it get us better participation? I just don't know how many people have the AWG Calendar embedded in their own calendar (that's the way I have it) because if they don't, there is a good possibility that a lot of people just don't know about the events we post on there.

@lin-fred @ewommack thoughts?

**Comment by ebraker** [comment]
@Jegelewicz Good catch. I didn't set these up, but I think it's a good idea to send out a monthly reminder for the office hour to the Google group. My guess is that many folks in the google group do not have the Arctos calendar embedded, so a monthly reminder will likely garner more participation (and I don't think it is too many emails personally).

**Comment by lin-fred** [comment]
I looked at the Arctos Calendar and the setting says public, but it also says it only shows busy/free status?
![image](https://user-images.githubusercontent.com/19802479/154367874-275d15fe-96c0-4ef1-9825-4ebc13cd819f.png)

There is another setting that says "see all details"
![image](https://user-images.githubusercontent.com/19802479/154367919-4f071691-23c9-4aa1-8961-841c35710fec.png)

Although a reminder might still be good, do we need to review what the calendar should be set to? As in, should it be set to public all details shown, and then people would know to make one private if it needs to be private

Or, does it need to stay as is and we just need to be more procedural with how public/private is set each time?

**Comment by Jegelewicz** [comment]
I think this would be an excellent topic for the Working Group Meeting!

**Comment by lin-fred** [comment]
And to confirm, @Jegelewicz when you say the Arctos Working Group Calendar, it's just the one Arctos Calendar?

**Comment by Jegelewicz** [comment]
Yeah, I think that's correct.

**Comment by lin-fred** [comment]
Well, my vote, is that it should be public/show all details. Because, if we are allowing people to add the calendar to their calendar, which gives them permission to make changes, then why are the details hidden by default?

**Comment by lin-fred** [comment]
or has this permission actually changed and people have to be invited now..

**Comment by Jegelewicz** [comment]
No idea! I don't know who actually owns the calendar....
